docs/release/support_matrix.md

```diff
@@ -1,6 +1,312 @@
+single high-speed network port for development and testing purposes,
+but this is not supported in a production environment.)
+It is strongly recommended that all DAOS engines in a DAOS system use the same
+model of high-speed fabric adapter.
+Heterogeneous adapter population across DAOS engines has **not** been tested,
+and running with such configurations may cause unexpected behavior.
+Please refer to "Fabric Support" below for more details.
+
+
+## Hardware platforms supported for DAOS Clients
+
+DAOS Version 2.2 supports the x86\_64 architecture.
+
+DAOS clients have no specific hardware dependencies.
+
+Each DAOS client needs a network port on the same high-speed interconnect
+that the DAOS servers are connected to.
+Multiple high-speed network ports per DAOS client are supported.
+Note that a single task on a DAOS client will always use a single network port,
+but when multiple tasks per client node are used then the DAOS agent will
+distribute the load by allocating different network ports to different tasks.
+
+
+## Operating Systems supported for DAOS Servers
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+DAOS Version 2.2 is primarily validated
+on [Rocky Linux 8.6](https://docs.rockylinux.org/release_notes/8.6/)
+and [openSUSE Leap 15.3](https://en.opensuse.org/openSUSE:Roadmap).
+The following subsections provide details on the Linux distributions
+which DAOS Version 2.2 supports on DAOS servers.
+
+Note that all DAOS servers in a DAOS server cluster (also called _DAOS system_)
+must run the same Linux distribution. DAOS clients that access a DAOS server
+cluster can run the same or different Linux distributions.
+
+### CentOS 7 and Red Hat Enterprise Linux 7
+
+With DAOS Version 2.2, CentOS 7.9 and RHEL 7.9 are supported on DAOS servers
+with 2nd gen Intel Xeon Scalable processors (Cascade Lake).
+
+CentOS 7.9 or RHEL 7.9 are **not** supported on DAOS servers
+with 3rd gen Intel Xeon Scalable processors (Ice Lake)
+or newer Intel Xeon processor generations.
+
+Links to CentOS Linux 7 and RHEL 7 Release Notes:
+
+* [CentOS 7.9.2009](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.2009)
+* [RHEL 7.9](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.9_release_notes/index)
+
+CentOS Linux 7 will reach End Of Life (EOL) on June 30st, 2024.
+Refer to the [RHEL Life Cycle](https://access.redhat.com/support/policy/updates/errata/)
+description on the Red Hat support website for information on RHEL support phases.
+
+### CentOS Linux 8
+
+CentOS Linux 8 has reached End Of Life (EOL) on December 31st, 2021.
+Consequently, DAOS Version 2.2 does not support CentOS Linux 8.
+DAOS clusters that have been running CentOS Linux 8 must be be migrated to
+a supported EL8 operating system in order to deploy DAOS Version 2.2.
+
+### Red Hat Enterprise Linux 8
+
+DAOS Version 2.2 is supported on RHEL 8.4 and RHEL 8.6. RHEL 8.5 is no longer supported by RedHat.
```

Comment:
Should this insist on RHEL 8.4EUS only or will we support out-of-date/unsupported 8.4 systems also?

Reply:
yes, limiting to EUS makes a lot of sense, fixing...